ice.c

```diff
@@ -339,6 +384,18 @@ static gboolean janus_ice_handles_check(gpointer user_data) {
 	}
 	janus_mutex_unlock(&old_handles_mutex);
 
+	if(janus_force_rtcpmux_blackhole_fd > 0) {
```

Comment:
janus_force_rtcpmux_blackhole_fd is initialized to `-1`, so if it is `>= 0` then it was set. Of course, it would only be zero if stdin was closed and no other fds took that number and stayed open, which is an impossible corner case, just thought I'd mention the very minor inconsistency :)